homeassistant/components/sensorpush_cloud/const.py

```diff
@@ -0,0 +1,30 @@
+"""Constants for the SensorPush Cloud integration."""
+
+from datetime import timedelta
+import logging
+from typing import Final
+
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN: Final = "sensorpush_cloud"
+MANUFACTURER: Final = "SensorPush"
+
+ATTR_ALTITUDE: Final = "altitude"
+ATTR_ATMOSPHERIC_PRESSURE: Final = "atmospheric_pressure"
+ATTR_BATTERY_VOLTAGE: Final = "battery_voltage"
+ATTR_BLUETOOTH: Final = "bluetooth"
+ATTR_DEWPOINT: Final = "dewpoint"
+ATTR_HUMIDITY: Final = "humidity"
+ATTR_LAST_UPDATE: Final = "last_update"
+ATTR_SIGNAL_STRENGTH: Final = "signal_strength"
+ATTR_VAPOR_PRESSURE: Final = "vapor_pressure"
+
+CONF_DEVICE_IDS: Final = "device_ids"
+
+ACCESS_TOKEN_EXPIRATION: Final = timedelta(minutes=60)
+
+UPDATE_INTERVAL: Final = timedelta(seconds=60)
+MAX_TIME_BETWEEN_UPDATES: Final = UPDATE_INTERVAL * 3
+
+REQUEST_RETRIES: Final = 3
+REQUEST_TIMEOUT: Final = 10  # seconds
```

Comment:
Maybe you can also use timedelta for this?

Reply:
I'm tempted. The swagger client is very particular about how timeouts are passed in. I could use a timedelta here as a form of documentation, and then convert to seconds in the API call itself. It adds a bit of code to api.py, but I don't have a strong opinion. What do you think?

Reply:
I gave this a shot and really like how it turned out. Thanks for the suggestion!